app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,23 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
```suggestion
      @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```
viết như này cho gọn e